docs/CaliptraSSIntegrationSpecification.md

```diff
@@ -820,6 +822,21 @@ Refer to the [MCU Veer-EL2 Core Configuration](../README.md#mcu-veer-el2-core-co
 
 Execute the full regression test suite documented in [How to test](#how-to-test) after any configuration changes to ensure system compatibility.
 
+## MCU DCCM SRAM Sizing
+
+MCU's DCCM SRAM should be sized large enough to accommodate FW's stack and heap. If it is undersized, the MCU would have to rely on the MCU SRAM (accessed via AXI) for the stack/heap which has much lower performance.
+
+## MCU iCache Integration Requirements
+
+The MCU's Memory Region Access Control (MRAC) regions are hard coded to 256MB boundaries. This means everything inside the 256MB boundary must be cachable in order to enable iCache. MCU SRAM lives inside MCI and not all regions of MCI are cachable. In order to enable MCU iCache the SOC must implement a split memory mapping scheme to separate the MCU SRAM from other MCI component: 
+
+
```

Comment:
Do we want to mention the mailbox SRAMs? They won't need to have I-Cache enabled, but if they are in the same region as MCI with side effect = 1, then they unaligned access will generate a hardware fault, which may be undesirable.